docs/gallery/static/impulse_response_measurement.ipynb

```diff
@@ -0,0 +1,500 @@
+    "\n",
+    "$$y[n] = h[n] \\ast x[n]\\,.$$\n",
+    "\n",
+    "The following sections will give an example of an impulse response measurement with an exponential sine sweep and a brief introduction of the related theory. For more background information see for example:\n",
+    "\n",
+    "Müller, S., & Massarani, P. (2001). Transfer function measurement with Sweeps. Directors cut including previously unreleased material and some corrections. *J. Audio Eng. Soc.* (Original Release), 49(6), 443–471.\n",
+    "\n",
+    "Before continuing with this example it is helpful to go through the `LINK-TO-SIGNALS-EXAMPLE` and `LINK-TO-ARITHMETICS`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import sounddevice as sd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Excitation signal\n",
+    "\n",
+    "Theoretically any signal can be used for excitation, however there are a few requirements that should be considered for acoustical measurements. The excitation signals should\n",
+    "\n",
+    "- be broad band, that is, it should cover the frequency range of interest for example the range of human hearing starting at 20 Hz and going up to 20 kHz\n",
+    "- contain enough energy to achieve a sufficient signal-to-noise ratio in the measured impulse response\n",
+    "- be robust to acoustic noise\n",
+    "- robust to harmonic distortion that can appear due to non-ideal sound sources (loudspeakers)\n",
+    "\n",
+    "Sine sweeps are sine signals with a continuously changing frequency that satisfy all these conditions. They are thus the predominant excitation signal for acoustical measurements. Their band width can be freely chosen and their energy can be adjusted by changing the length of the sweep signal.\n",
+    "\n",
+    "This example uses an exponential sine sweep designed in the time domain. For illustration purposes, the bandwidth is restricted between 50 Hz and 16 kHz. The sweep can be generated with the [`pyfar.signals`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.signals.html) module"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = pf.signals.exponential_sweep_time(\n",
+    "    n_samples=2**16,\n",
+    "    frequency_range=[50, 22050],\n",
+    "    sampling_rate=44100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and plotted with the [`pyfar.plot`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.plot.html) module"
```

Comment:
Link is not rendered correctly at https://pyfar-gallery.readthedocs.io/en/enhancement-measurement_nb_static/gallery/static/impulse_response_measurement.html